spec.html

```diff
@@ -4309,7 +4306,7 @@ <h1>IsGenericDescriptor ( _Desc_ )</h1>
 
       <emu-clause id="sec-frompropertydescriptor" aoid="FromPropertyDescriptor">
         <h1>FromPropertyDescriptor ( _Desc_ )</h1>
-        <p>When the abstract operation FromPropertyDescriptor is called with Property Descriptor _Desc_, the following steps are taken:</p>
+        <p>The abstract operation FromPropertyDescriptor takes argument _Desc_ (a Property Descriptor or *undefined*). It performs the following steps:</p>
```

Comment:
I notice that in some places, we have explicit asserts for any preamble-declared types, but eg in this one, we don't. Is that an example of the inconsistency you don't want to handle in this PR?

Reply:
> Is that an example of the inconsistency you don't want to handle in this PR?

Yes. I agree those would also be good to address, I just don't want to consider them in scope for this.